x-pack/legacy/plugins/siem/server/lib/timeline/types.ts

```diff
@@ -204,21 +208,36 @@ export const AllTimelineSavedObjectRuntimeType = runtimeTypes.type({
 export interface AllTimelineSavedObject
   extends runtimeTypes.TypeOf<typeof AllTimelineSavedObjectRuntimeType> {}
 
-export interface ExportTimelineRequestParams {
-  body: { ids: string[] };
-  query: {
-    file_name: string;
-    exclude_export_details: boolean;
-  };
+/**
+ * Import/export timelines
+ */
+
+export interface ExportTimelinesQuery {
```

Comment:
This is redundant with `t.TypeOf<ExportTimelinesQuerySchema>` and can be deleted.